lib/core/extension/navigation_extension.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class NavigationExtension {
```

Comment:
should be change name class
```suggestion
class NavigationManager {
```